common/src/main/resources/tachyon-default.properties

```diff
@@ -116,17 +114,17 @@ tachyon.user.heartbeat.interval.ms=1000
 tachyon.user.file.writetype.default=CACHE_THROUGH
 tachyon.user.file.tachyonstoragetype.default=STORE
 tachyon.user.file.understoragetype.default=SYNC_PERSIST
-tachyon.user.default.block.size.byte=512MB
+tachyon.user.block.size.bytes.default=512MB
```

Comment:
Yes, sorting them lexicographically is my plan too. But I'd like to leave that in a followup PR with some other mechanical changes.